change/@fluentui-web-components-fe13c853-dc9c-49fc-92e0-cd66ead7a217.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32075&commitId=1947fef21f2fd8d87718b6b4dc1c1f5ba8bd460b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons Dark Mode.chromium.png|1422422| Changed|
|Accordion.Size.chromium.png|1603062| Changed|
|Accordion.Accordion With Custom Icons.chromium.png|1608095| Changed|
|Accordion.Size Dark Mode.chromium.png|1244339| Changed|
|Accordion.Marker Position End RTL.chromium.png|1609531| Changed|
|Accordion.Accordion With Custom Icons RTL.chromium.png|1608070| Changed|
|Accordion.Disabled RTL.chromium.png|1609301| Changed|
|Accordion.Marker Position End Dark Mode.chromium.png|1461050| Changed|
|Accordion.Marker Position End.chromium.png|1609559| Changed|
|Accordion.Disabled.chromium.png|1609413| Changed|
|Accordion.Size RTL.chromium.png|1602956| Changed|
|Accordion.Disabled Dark Mode.chromium.png|1461116| Changed|

</details>


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>